src/services/ui/src/pages/actions/WithdrawRai.tsx

```diff
@@ -1,224 +1,44 @@
+      intro={
+        <ActionFormIntro title={"Withdraw Formal RAI Response Details"}>
+          <I.RequiredIndicator /> Indicates a required field
+          <p className="font-light mb-6 max-w-4xl">
+            Complete this form to withdraw the Formal RAI response. Once
+            complete, you and CMS will receive an email confirmation.
           </p>
-          {attachmentList.map(({ name, label, required }) => (
-            <I.FormField
-              key={name}
-              control={form.control}
-              name={`attachments.${name}`}
-              render={({ field }) => (
-                <I.FormItem>
-                  <I.FormLabel>
-                    {label}
-                    {required ? <I.RequiredIndicator /> : ""}
-                  </I.FormLabel>
-                  <I.Upload
-                    files={field?.value ?? []}
-                    setFiles={field.onChange}
-                  />
-                  <I.FormMessage />
-                </I.FormItem>
-              )}
-            />
-          ))}
-          <I.FormField
-            control={form.control}
-            name="additionalInformation"
-            render={({ field }) => {
-              return (
-                <I.FormItem>
-                  <h3 className="font-bold text-2xl font-sans">
-                    Additional Information
-                    <I.RequiredIndicator />
-                  </h3>
-                  <I.FormLabel className="font-normal">
-                    Explain your need for withdrawal.
-                  </I.FormLabel>
-                  <I.Textarea
-                    {...field}
-                    value={field.value || ""}
-                    className="h-[200px] resize-none"
-                  />
-                  <I.FormDescription>
-                    4,000 characters allowed
-                  </I.FormDescription>
-                </I.FormItem>
-              );
-            }}
-          />
-          {form.formState.isSubmitting && (
-            <div className="p-4">
-              <LoadingSpinner />
-            </div>
-          )}
-          <div className="flex gap-2">
-            <I.Button
-              disabled={form.formState.isSubmitting}
-              type="button"
-              onClick={async () => {
-                await form.trigger();
-
-                if (form.formState.isValid) {
-                  setAreYouSureModalIsOpen(true);
-                }
-              }}
-              className="px-12"
-            >
-              Submit
-            </I.Button>
-            <I.Button
-              type="button"
-              variant="outline"
-              onClick={() => setCancelModalOpen(true)}
-              className="px-12"
-            >
-              Cancel
-            </I.Button>
-          </div>
-        </form>
-      </I.Form>
-      {/* Are you sure modal */}
-      <ConfirmationModal
-        open={areYouSureModalIsOpen}
-        onAccept={() => {
```

Comment:
This modal has been removed _**FOR NOW**_. We have a documented requirement from HCD for modals on destructive actions (namely the two withdraw actions), however, there's a pattern to be figured out between `react-hook-form` and the modal control flow that would further hold up this PR and thus tooling that helps continued work on Waiver submissions and actions.

Reply:
I've also let Padma know this regarding testing.

Reply:
Comment from JIRA: https://qmacbis.atlassian.net/browse/OY2-26239?focusedCommentId=127105